templates/blocks/share_buttons.twig

```diff
@@ -37,6 +37,15 @@
             <span class="visually-hidden">{{ __( 'Share via', 'planet4-master-theme' ) }} Email</span>
         </a>
     {% endif %}
+    <!-- Bluesky -->
+    {% if share_platforms.bluesky %}
+        <a href="https://bsky.app/intent/compose?text={{ social.title|url_encode }}{% if share_text or social.description %} - {{ (share_text ?? social.description)|striptags|url_encode }}{% endif %}, via @{{ social_accounts.bluesky ?? 'greenpeace' }} {{ (socialLink ~ '&utm_source=bluesky')|url_encode }}"
+           onclick="window.dataLayerPush('Bluesky');"
+           target="_blank" class="share-btn bluesky">
+            {{ 'bluesky'|svgicon }}
+            <span class="visually-hidden">{{ __( 'Share on', 'planet4-master-theme' ) }} Bluesky</span>
```

Comment:
I'm wondering if this might lead to translation issues in some languages? 🤔 Maybe in some cases "Bluesky" would not be at the end of the sentence for example... it could be better to use a variable? I know it's the same for other platforms though so it can be handled in a follow-up